smtp/init.lua

```diff
@@ -155,27 +162,65 @@ curl_mt = {
             if not body or not url or not from then
                 error('request(url, from, to, body [, options]])')
             end
-            local header = ''
+            local header
             local recipients = {}
             header = 'From: ' .. from .. '\r\n' ..
                      'To: ' .. add_recipients(recipients, to) .. '\r\n'
             if opts.cc then
                 header = header .. 'Cc: ' .. add_recipients(recipients, opts.cc) .. '\r\n'
             end
+            if opts.subject then
+                header = header .. 'Subject: ' .. opts.subject .. '\r\n'
+            end
             add_recipients(recipients, opts.bcc)
             if opts.headers and #opts.headers > 0 then
                 header = header .. table.concat(opts.headers, '\r\n') .. '\r\n'
             end
+            local content_type = 'Content-Type: ' .. (opts.content_type or 'text/plain') ..
+            '; charset=' .. (opts.charset or  'UTF-8') ..';\r\n'
```

Comment:
Nit: This header is added only when there are no attachments, so I would sink it down to the corresponding if-branch.

Reply:
I'm disagree with your point cause 'content_type' is alo used in "with attachments" clause of if, see: lines 216-223 of init.lua
Generally in case of multipart bode content we still need point body content-type as it is a part of of multipart and every single part have to have content_type.

Reply:
I overlooked it. Agreed.